src/Minter/MinterAPI.php

```diff
@@ -167,12 +167,19 @@ public function getCoinInfo(string $coin): \stdClass
      * Returns block data at given height.
      *
      * @param int $height
+     * @param bool $withEvents
      * @return \stdClass
      * @throws \Exception
      */
-    public function getBlock(int $height): \stdClass
+    public function getBlock(int $height, $withEvents = false): \stdClass
     {
-        return $this->get('/api/block/' . $height);
+        $uri = '/api/block/' . $height;
+
+        if ($withEvents === true) {
+            $uri .= '?withEvents=true';
+        }
+
+        return $this->get($uri);
```

Comment:
Please make the return response in one line. Like this
`return $this->get('/api/block/' . $height . ($withEvents ? '?withEvents=true' : ''))`